src/cards/meteor.ts

```diff
@@ -133,11 +139,13 @@ async function meteorProjectiles(meteorLocations: Vec2[], underworld: Underworld
     makeForceMoveProjectile({
       pushedObject,
       startPoint: startPos,
-      endPoint: meteor.destination,
-      speed: distanceOffset / meteor.travelTime,
-      doesPierce: true,
-      ignoreUnitIds: underworld.units.map(u => u.id),
+      velocity,
+      piercesRemaining: 10000,
```

Comment:
Use `Infinity` which is an actual number in js.  That way subtracting from it will not decrease it